login.js

```diff
@@ -0,0 +1,13 @@
+let password = document.getElementById('password_field').value;
+const output = document.getElementById('validation');
+
+document.getElementById('submit')
+.onclick = function (password) {
+  if (password === '12345678') {
+    output.textContent = 'Welcome Back!';
+    return;
```

Comment:
you don't need this return, you have an `else`, so if the condition in the `if` is not fulfilled, it will jump to the `else`.